### PR TITLE
Testing To Do list: part 2

### PR DESCRIPTION
**Pull request by anuarshaidenov**
## Milestone details:
- Created a test for updating the status.
- Created a test for the clear all tasks function.
- Created a test for editing the tasks.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test/list.test.js [hunk 1]
```diff
@@ -47,4 +48,14 @@ describe('Check add and remove functionality', () => {
     // Remove test
     expect(todoList.list).toHaveLength(2);
   });
+
+  describe('Updating a task', () => {
+    test('Update task status', () => {
+      const todoList = new List();
+      todoList.addActivity('Test');
+      todoList.updateActivityStatus(1);
```
Comment (mikenath223) [inline test/list.test.js:56]: - [x] Kindly add a test for the Clear All completed functionality.

Comment (anuarshaidenov) [reply test/list.test.js:56]: @mikenath223 Thank you for the feedback, Mike. We've made some changes I think we should be good now🙂

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @KayLemba and @anuarshaidenov, 

Your project is complete!  There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 

Great job, keep going

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**